src/sentry/api/serializers/models/project.py

```diff
@@ -679,14 +679,19 @@ def get_attrs(
             deploys_by_project = self.get_deploys_by_project(item_list)
 
         for item in item_list:
+            # check if the project is in LPQ for any platform
+            symbolication_degraded = False
+            for platform in ["native", "js", "jvm"]:
```

Comment:
This for loop can be simplified by using any(x == b for x in y) syntax

Reply:
Good call, it's much nicer this way.